SUMMARY.md

```diff
@@ -1,3 +1,9 @@
 # Table of contents
 
 * [Introduction](README.md)
+
+* [Warning](SUMMARY.md)
+
+{% hint style="warning" %}
+**A Hint** I am a hint.
+{% endhint %}
```

Comment:
This part here, needs to be in a new markdown file. Not in `SUMMARY.md`

Reply:
Updated